diff -Naur kylin-recorder-1.2.23/src/mainwindow.cpp kylin-recorder-1.2.23~/src/mainwindow.cpp
--- kylin-recorder-1.2.23/src/mainwindow.cpp	2021-06-17 14:46:25.000000000 +0800
+++ kylin-recorder-1.2.23~/src/mainwindow.cpp	2021-09-14 11:53:11.591386995 +0800
@@ -136,6 +136,7 @@
     mainWid ->setFixedSize(WIDTH,HEIGHT);
 //    mainWid ->setWindowIcon(QIcon::fromTheme("kylin-recorder", QIcon(":/svg/svg/recording_128.svg")));
     mainWid ->setWindowTitle(tr("Recorder"));
+    mainWid->setWindowFlags(Qt::FramelessWindowHint | Qt::Widget);
     //屏幕中间老方法
     //    this->move((QApplication::desktop()->width() -WIDTH)/2, (QApplication::desktop()->height() - HEIGHT)/2);
     //显示在活动屏幕中间新方法
